wiki/Shortcut_key_reference/en.md

```diff
@@ -131,6 +131,18 @@ These shortcuts work anywhere:
 | `F3` or `F4` | Adjust speed for osu!mania (can be used near the beginning of gameplay). |
 | `Ctrl` + `R` or `` ` `` | Must be held briefly. Quick retry the current map (skips the intro upon retry) |
 
+## Replay mode
+
+*Notice: You must use the cuttingedge release stream to use these.*
```

Comment:
if this type of thing is supposed to be a `Notice` I think I've been mistakenly calling them `Note`s all this time 😨

Reply:
i believe "note" is more correct.

"note that"

Reply:
https://osu.ppy.sh/help/wiki/Article_Styling_Criteria#notices

We use other words to express how important it is and give attention to.

Reply:
Isn't this the exact same as NB: